test-site/src/pages/PeoplePage.tsx

```diff
@@ -7,10 +8,12 @@ import {
   StandardCard,
   VerticalResults,
   LocationBias,
-  StaticFilters
+  StaticFilters,
+  Facets,
+  ApplyFiltersButton
 } from '@yext/answers-react-components';
-import { useLayoutEffect } from 'react';
-import { Facets, hierarchicalFacetFieldIds } from '../components/Facets';
+
+export const hierarchicalFacetFieldIds = ['c_hierarchicalFacet'];
```

Comment:
why is this exported?

Reply:
oops, copied over from the all Facets component in test-site and forgot to remove export